Themis/Models/Assessment.swift

```diff
@@ -54,7 +60,7 @@ class AssessmentResult: Encodable, ObservableObject {
 
     func encode(to encoder: Encoder) throws {
         var container = encoder.container(keyedBy: CodingKeys.self)
-        try container.encode(score, forKey: .score)
+        try container.encode(relativeScore, forKey: .score)
```

Comment:
I would add a comment here to indicate that this is intended behavior and not a bug (it might look a bit weird if future developers look at this).

Reply:
With the renamed variables it should be clear now